apps/hestia/src/components/ui/ConnectWallet/insufficientBalance.tsx

```diff
@@ -13,10 +12,20 @@ import { MINIMUM_PDEX_REQUIRED } from "@orderbook/core/constants";
 import { useState } from "react";
 import { getChainFromTicker, useAssets } from "@orderbook/core/index";
 import Link from "next/link";
+import classNames from "classnames";
 
 import { GenericInfoCard, GenericExternalCard } from "../ReadyToUse";
 import { Icons } from "..";
 
+const filteredAssets = [
+  "222121451965151777636299756141619631150",
+  "95930534000017180603917534864279132680",
+  "226557799181424065994173367616174607641",
+  "3496813586714279103986568049643838918",
+];
+
+const activeAssets = ["95930534000017180603917534864279132680"];
```

Comment:
These variables`filteredAssets` and `activeAssets` should be dynamic in my opinion. It doesn't make sense to provide asset Id here.